_tutorials/new-component/create-a-new-component-part-2.md

```diff
@@ -59,13 +61,17 @@ Let's break this down a little.
 ## Imports
 
 The first line imports Sass from `src/scss/_variables.scss`. Note the underscore and extension is not needed in the `@import` statement. All Origami Sass files except `main.scss` should start with an underscore to indicate they are [Sass partials](https://sass-lang.com/documentation/at-rules/import#partials) for import and should not be compiled on their own.
-<pre><code class="o-syntax-highlight--scss">@import 'src/scss/variables';</code></pre>
+<pre><code class="o-syntax-highlight--scss">// main.scss
```

Comment:
I really like that you've done the file path location in a comment syntax which matches the language of the file <3

Reply:
except origami.json 🙈